### PR TITLE
Fix Clujbike

### PR DESCRIPTION
**Pull request by martgnz**
Endpoint shut down. Loads data from this map instead: https://portal.clujbike.eu/harta-statii-biciclete

### CODE REVIEW

File: pybikes/clujbike.py [hunk 1]
```diff
@@ -19,70 +20,34 @@ def __init__(self, tag, feed_url, meta):
...
-            'IdStatus' : '0',
-            'Status' : '',
-            'StatusType' : '',
-            'Latitude' : '0',
-            'Longitude' : '0',
-            'IsValid' : 'true',
-            'CustomIsValid' : 'false',
-            'Id' : '0',
-        }
+        data = re.search(r'var stations = (.*?;)', raw).group(1).replace('}];', '}]')
```
Comment (eskerda) [inline pybikes/clujbike.py:27]:
```suggestion
        data =  re.search(r'var stations = (\[.*?\]);' raw)
```

Comment (martgnz) [reply pybikes/clujbike.py:27]: thank you!!!

File: pybikes/clujbike.py [hunk 2]
```diff
@@ -19,70 +20,34 @@ def __init__(self, tag, feed_url, meta):
...
+        for station in json.loads(data):
+            stations.append(ClujBikeStation(station))
 
         self.stations = stations
+
+
+class ClujBikeStation(BikeShareStation):
+    def __init__(self, data):
+        super(ClujBikeStation, self).__init__()
+        print(data)
```
Comment (eskerda) [inline pybikes/clujbike.py:40]:
```suggestion
```

Comment (martgnz) [reply pybikes/clujbike.py]: sorry, very embarrasing

File: pybikes/clujbike.py [hunk 3]
```diff
@@ -19,70 +20,33 @@ def __init__(self, tag, feed_url, meta):
...
+
+        self.name = data['StationName']
+        self.latitude = float(data['Latitude'])
+        self.longitude = float(data['Longitude'])
+
+        self.bikes = int(data['OcuppiedSpots'])
+        self.free = int(data['EmptyDoors'])
+        self.status = 'offline' if data['Status'] == 'Offline' else 'online'
+        self.extra = {
+            'slots': self.bikes + self.free,
```
Comment (eskerda) [inline pybikes/clujbike.py:49]: Not sure we can trust to properly represent the number of slots that stations have. I see there are many stations with 19 slots. I would go with removing it unless it's constantly accurate.

Comment (martgnz) [reply pybikes/clujbike.py]: Sure, just thought it could be helpful. Removed!

**Pull request merged**